.eslintrc

```diff
@@ -1,5 +1,5 @@
 rules:
-    indent: 4
+    "indent": 2
```

Comment:
Grabbed this from the eslint docs. Some of their rules show example usage with quotes, some not. I have just tested this locally and it doesn't appear that quoting effects anything. I've unquoted this to be consistent.